apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetService.java

```diff
@@ -51,12 +51,16 @@
 import static com.comet.opik.domain.ExperimentItemDAO.ExperimentSummary;
 import static com.comet.opik.infrastructure.db.TransactionTemplateAsync.READ_ONLY;
 import static com.comet.opik.infrastructure.db.TransactionTemplateAsync.WRITE;
+import static jakarta.ws.rs.core.Response.Status.CONFLICT;
```

Comment:
import static jakarta.ws.rs.core.Response, is already imported, and the convention in the different java files.

We could call `Response.Status.CONFLICT` and `Response.Status.NOT_FOUND`

Reply:
Didn't quite understand what you mean here. Please explain.